File: web-app/src/app/services/types/Companytechonology.ts
import { Companydatum } from "./Companydatum";
import { Technology } from "./Technology";

export type Companytechonology = {
    Id: number;
    CompanyId?: number;
    TechnologyId?: number;
    IsDeleted: boolean;
    Company: Companydatum[];
    Technology: Technology[];
}

File: Placements.DataAccess/Placement/Models/Companytechonology.cs
using System;
using System.Collections.Generic;

namespace Placements.DataAccess.Placement.Models;

public partial class Companytechonology
{
    public long Id { get; set; }

    public long? CompanyId { get; set; }

    public long? TechnologyId { get; set; }

    public ulong IsDeleted { get; set; }

    public virtual Companydatum? Company { get; set; }

    public virtual Technology? Technology { get; set; }
}
